qa/tasks/scripts/ses_stop_osd_daemon.sh

```diff
@@ -7,7 +7,7 @@ stopped_osd=()
 for i in ${!storage_minions[@]}
 do
     minion=${storage_minions[i]%%.*}
-    random_osd=$(ceph osd tree | grep -A 1 $minion | grep -o osd.* | awk '{print $1}')
+    random_osd=$(ceph osd tree | grep -A 1 $minion | grep -Eo "osd\.[^[:space:]]+")
```

Comment:
Do you want to match strings like `osd.foo` ? Or just `osd.5`?

```
smithfarm@wilbur:~/scratch> grep -Eo "osd\.[^[:space:]]+" bubba
osd.foo
osd.0
osd.bar
osd.12
smithfarm@wilbur:~/scratch> grep -Eo "osd\.[[:digit:]]+" bubba
osd.0
osd.12
```

Reply:
Thanks.